includes/settings.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
@AkankshaBodhankar  I have created a separate file for google reCaptcha credentials for client and server authentication, it cant be included in the same file as it exposes the risk of key exposure.  Whenever a new user has to set up project for the first time, he needs to generate his /her key and add it to the file. We need to add this to the documentation as well, mentioned in #157 

Posting screenshot in a while.